packages/grid/x-data-grid-premium/src/hooks/features/clipboard/useGridClipboardImport.ts

```diff
@@ -301,11 +301,7 @@ function defaultPasteResolver({
 }
 
 function isPasteShortcut(event: React.KeyboardEvent) {
-  const isModifierKeyPressed = event.ctrlKey || event.metaKey || event.altKey;
```

Comment:
`event.altKey` can't be used to trigger the paste shortcut. It's a macOS-only key that never pastes: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/altKey. Related to https://github.com/mui/mui-x/pull/7466.